browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1773,7 +1773,12 @@ L.CanvasTileLayer = L.Layer.extend({
 			this._onCalcFunctionListMsg(textMsg.substring('calcfunctionlist:'.length + 1));
 		}
 		else if (textMsg.startsWith('tooltip:')) {
-			this._onCalcFunctionUsageMsg(textMsg.substring('tooltip:'.length + 1));
+			if (textMsg.startsWith('tooltip: formulausage:'))
```

Comment:
We do core change anyway. Can we simplify that and just use JSON always? That will also prevent us from unexpected new types of tooltips.

```
var tooltipInfo = JSON.parse(textMsg.substring('tooltip:'.length + 1));
if (tooltipInfo.type === 'formulausage')
    onCalcFunctionUsage(tooltipInfo.text);
else if (tooltilInfo.type === 'trackchange')
   showDocumentTooltip(tooltipInfo);
else
    console.error('unknown tooltip type');
```

Reply:
sure, thanks